iocage_lib/ioc_create.py

```diff
@@ -359,37 +389,38 @@ def _create_jail(self, jail_uuid, location):
                 start = False
                 is_template = True
 
-            try:
-                value, config = iocjson.json_check_prop(key, value, config)
-
-                config[key] = value
-            except RuntimeError as err:
-                iocjson.json_write(config)  # Destroy counts on this.
-                iocage_lib.ioc_destroy.IOCDestroy().destroy_jail(location)
-
-                raise RuntimeError(f"***\n{err}\n***\n")
-            except SystemExit:
-                iocjson.json_write(config)  # Destroy counts on this.
-                iocage_lib.ioc_destroy.IOCDestroy().destroy_jail(location)
-                exit(1)
+                try:
+                    iocjson.json_check_prop(key, value, config)
```

Comment:
Why the removal? We want value and config returned here. For instance `ip4_addr` will possibly be `DEFAULT` and the user will want a default gateway set.